frontend/src/pages/policies/PrivacyConsent.tsx

```diff
@@ -0,0 +1,89 @@
+import { ReactComponent as Logo } from '@assets/icons/logo.svg';
+import { theme } from '../../styles/theme';
+import * as S from './PolicyPage.common.styles';
+
+const PrivacyConsent = () => {
+  return (
+    <S.Wrapper>
+      <S.Header>
+        <Logo fill={theme.colors.gray06} />
+        <S.Title>개인정보 수집 및 이용 동의</S.Title>
+      </S.Header>
+
+      <S.Content>
+        <S.Section>
+          <S.Paragraph>
+            포게더(이하 "회사")는 「개인정보보호법」 등 관련 법령에 따라
+            회원님의 개인정보를 안전하게 관리하며, 아래와 같은 목적으로
+            개인정보를 수집·이용합니다. 회원님께서는 동의를 거부할 권리가
+            있으며, 동의를 거부할 경우 서비스 이용에 제한이 있을 수 있습니다.
+          </S.Paragraph>
+        </S.Section>
+        <S.Section>
+          <S.SectionTitle>1. 개인정보 수집 및 이용 목적</S.SectionTitle>
+          <S.List>
+            <S.ListItem>
+              서비스 제공: 사진 공유 스페이스 생성 및 관리, 사진
+              업로드/다운로드, 회원 식별
+            </S.ListItem>
+            <S.ListItem>
+              서비스 개선: 서비스 이용 통계 분석 및 사용자 경험 개선
+            </S.ListItem>
+            <S.ListItem>
+              보안 및 이용자 보호: 부정 이용 방지, 서비스 안정성 확보
+            </S.ListItem>
+          </S.List>
+        </S.Section>
+
+        <S.Section>
+          <S.SectionTitle>2. 수집하는 개인정보 항목</S.SectionTitle>
+          <S.Paragraph>필수항목</S.Paragraph>
+          <S.List>
+            <S.ListItem>스페이스 생성 시 입력한 이름</S.ListItem>
+            <S.ListItem>카카오 소셜 로그인 시: 닉네임</S.ListItem>
+          </S.List>
+          <S.Paragraph>선택항목</S.Paragraph>
+          <S.List>
+            <S.ListItem>카카오 소셜 로그인 시: 프로필 사진 </S.ListItem>
```

Comment:
아주 사소하지만 공백 한칸 제거하면 어떨까요!

```suggestion
            <S.ListItem>카카오 소셜 로그인 시: 프로필 사진</S.ListItem>
```

Reply:
아니 어덯게 발견하셨죠 ㄷㄷ 수정하였습니다 ~
[a17b875](https://github.com/woowacourse-teams/2025-PhotoGather/pull/402/commits/a17b87567ce83e3b84e4d93b86087fe9d2c27851)